ReIndexOrchestrator/index.js

```diff
@@ -0,0 +1,44 @@
+const df = require('durable-functions');
+const moment = require('moment');
+
+module.exports = df.orchestrator(function* orchestratorFunctionGenerator(context) {
+  context.log('Starting ReIndexOrchestrator');
+  const input = context.df.getInput();
+
+  context.log({ input });
+
+  const indexNames = input.parameters.indexNames;
+  const apimApiName = input.parameters.apimApiName;
+  let indexDefinition = input.parameters.indexDefinition;
+
+  if (!indexDefinition) {
+    indexDefinition = yield context.df.callActivity('GetIndexDefinition', indexNames.active);
+  }
+  context.log(indexDefinition);
+
+  const indexerName = yield context.df.callActivity('ReIndex', { indexDefinition, indexNames });
+  context.log({ indexerName });
```

Comment:
Could some additional information be added to increase the usefulness of this message within the logs please?